### ISSUE TITLE
Improved controls

### ISSUE THREAD

**Issue by yellowberryHN** [DreamBlocks/DreamBlocks#78]
At its current state, the cursor controls are very confusing and difficult to use. It seems like it points in the opposite direction I want it to, all the time.

**Comment by LordStephen77** [comment]
@YellowberryHN Yes you are right. The project is undergoing improvement. As soon as I can finish the new release with all the fixes and improvements the audience. The pointers originally served during debugging, then left. In the current version the pointer is only one and serves as a box-selector.

Thanks for your advice.
